src/main/scala/dsptools/counters/Counters.scala

```diff
@@ -66,21 +66,21 @@ class CountIO (countParams: CountParams) extends Bundle {
   // Count up/down control signal
   val upDown = if (countParams.countType == UpDown) Some(Input(Bool())) else None
   // Counters usually increment by 1
-  val inc = if (countParams.incMax != 1) Some(Input(UInt(countParams.incMax))) else None
```

Comment:
Again, in Chisel2, I never wanted IO Bundles. I just wanted to be able to separately specify whether each signal was an IO or not, so I hacked my way around it, because no one wanted to support it.